src/middlewared/middlewared/plugins/account.py

```diff
@@ -777,8 +777,6 @@ def do_update(self, app, audit_callback, pk, data):
             verrors.add('user_update.smb',
                         'Password must be reset in order to enable SMB authentication')
 
-        verrors.check()
```

Comment:
Validation must occur before we make on-disk changes. This breaks our validation logic.